.github/workflows/tests.yml

```diff
@@ -41,6 +41,11 @@ jobs:
         run: |
           python -m pip install flake8==5.0.4
           python -m flake8 spacy --count --select=E901,E999,F821,F822,F823,W605 --show-source --statistics
+      - name: cython-lint
+        run: |
+          python -m pip install cython-lint -c requirements.txt
+          cython-lint spacy
```

Comment:
hey, thanks for using cython-lint

if the changes are too many, you might want to start by enabling it with `--no-pycodestyle`? that at least is what arrow did

Reply:
Thanks for the hint!